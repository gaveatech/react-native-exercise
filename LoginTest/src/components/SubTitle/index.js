import React from 'react';
import styled from 'styled-components/native';
import PropTypes from 'prop-types';

const SubTitle = ({children}) => {
  return <Text>{children}</Text>;
};

const Text = styled.Text`
  font-family: 'RobotoLight';
  font-size: 16px;
  margin-bottom: 20px;
`;

SubTitle.propTypes = {
  children: PropTypes.node.isRequired,
};

export default SubTitle;
