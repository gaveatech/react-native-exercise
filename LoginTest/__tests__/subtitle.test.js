import React from 'react';
import renderer from 'react-test-renderer';
import SubTitle from '../src/components/SubTitle';

describe('<SubTitle>', () => {
  it('Snapshot', () => {
    const tree = renderer.create(<SubTitle>Teste</SubTitle>).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
