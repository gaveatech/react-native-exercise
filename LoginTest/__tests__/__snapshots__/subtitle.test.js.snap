// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SubTitle> Snapshot 1`] = `
<Text
  style={
    Array [
      Object {
        "fontFamily": "RobotoLight",
        "fontSize": 16,
        "marginBottom": 20,
      },
    ]
  }
>
  Teste
</Text>
`;
